segmenti/segmenti.py

```diff
@@ -26,47 +28,47 @@ def sostenitori_cri(queryset):
 
 
 def aspiranti_volontari_iscritti_ad_un_corso(queryset):
-    pass
+    return queryset.filter(aspirante__isnull=False, partecipazioni_corsi__isnull=False)
 
 
 def tutti_i_presidenti(queryset):
-    pass
+    return queryset.filter(delega__tipo=PRESIDENTE)
 
 
 def presidenti_comitati_locali(queryset):
-    pass
+    return tutti_i_presidenti(queryset).filter(appartenenze__sede__tipo=Sede.COMITATO, appartenenze__sede__estensione=LOCALE)
 
 
 def presidenti_comitati_regionali(queryset):
-    pass
+    return tutti_i_presidenti(queryset).filter(appartenenze__sede__tipo=Sede.COMITATO, appartenenze__sede__estensione=REGIONALE)
 
 
 def delegati_US(queryset):
-    pass
+    return queryset.filter(delega__tipo=UFFICIO_SOCI)
 
 
 def delegati_Obiettivo_I(queryset):
-    pass
+    return queryset.filter(delega__tipo=DELEGATO_OBIETTIVO_1)
```

Comment:
Per quanto riguarda le deleghe, controllare il tipo va bene, ma `Delega` come qualunque entita' col Mixin `ConStorico` ha data di inizio e data di fine, quindi bisogna filtrare per quelle attuali.

Assumendo che `queryset` sia un QuerySet di oggetti `Persona`, questo potrebbe diventare:

```python
  return queryset.filter(Delega.query_attuale(tipo=DELEGATO_OBIETTIVO_1).via("deleghe"))
```

Il `query_attuale` e' un metodo "concept" che viene risolto a un oggetto Q contenente tutte le condizioni per ottenere le deleghe attuali che abbiano tipo desiderato. Il `.via` permette aggiungere il prefisso, in quanto si sta risolvendo un queryset per Persona (ove `deleghe` e' il `related_name` di `Delega.persona`).